core/templates/services/schema-default-value.service.ts

```diff
@@ -119,6 +124,9 @@ export class SchemaDefaultValueService {
     );
 
     if ('choices' in schema) {
+      if (schema.choices === undefined) {
+        throw new Error('Schema Choices does not exist');
+      }
```

Comment:
Can these two ifs be merged together?

Reply:
No, because we don't want to throw an error if schema.choices is defined

Reply:
But when `'choices' in schema` is true then we can be sure that `schema.choices === undefined` will be false, right?

Reply:
ah ok ,done